### PR TITLE
Wizard JMX Metrics form

### PR DESCRIPTION
**Pull request by David-DB88**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
created JMX Metrics form
**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Wizard/WizardForm/WizardForm.tsx [hunk 1]
```diff
@@ -46,44 +46,36 @@ const Wizard: React.FC<WizardFormProps> = () => {
...
+          <div>
+            <Button buttonSize="M" buttonType="primary">
+              Add Kafka Connect
+            </Button>
+          </div>
+        </S.Section>
+        <JMXMetrics />
+        <hr />
+        <div style={{ paddingTop: '10px' }}>
+          <div
```
[the remark this answers is not in the document]
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Wizard/WizardForm/WizardForm.tsx]: yes we have a ButtonWrapper i changed it

File: kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            truststorePassword: '',
+            keystoreLocation: '',
+            keystorePassword: '',
+            keystoreKeyPassword: '',
+            isAuth: '',
+            username: '',
+            password: '',
+          },
+        },
+        { keepDefaultValues: true }
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx:34]: please stop resetting form& Just use setValue

File: kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  const [newMetrics, setNewMetrics] = useState(false);
+  const { reset, getValues, watch } = useFormContext();
+  const showMetrics: React.MouseEventHandler<HTMLButtonElement> = (e) => {
+    e.preventDefault();
+    setNewMetrics(!newMetrics);
+    if (!newMetrics) {
+      reset(
+        {
+          ...getValues(),
+          Metrics: {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx:21]: uppercase?

File: kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        </Button>
+      </div>
+      {newMetrics && (
+        <>
+          <ControlledSelect
+            name="Metrics.type"
+            label="Metrics Type"
+            options={METRICS_OPTIONS}
+          />
+          <div style={{ width: '20%' }}>
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx:58]: styled components

**Review by workshur — CHANGES_REQUESTED** [review]

File: kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx (hunk 2 above)
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx:34]: ok

File: kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx (hunk 3 above)
Comment (David-DB88) [reply kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx:21]: ok i changed, it's because in a schema it wrote uppercase

**Review by workshur — APPROVED** [review]

**Pull request merged**